### ISSUE TITLE
Webhook recevied action to allow for status updates to be forwarded.

### ISSUE THREAD

**Issue by gregrobsonoxb** [dividohq/finance-plugin-woocommerce#73]
**Is your feature request related to a problem? Please describe.**
Currently we have orders that are placed online. We need any online application status updates to be fed into our CRM. As applications have only one webhook URL and there isn't a "global" way to collection application details I have to patch each plugin update after [this line](https://github.com/dividohq/finance-plugin-woocommerce/blob/master/class-wc-gateway-finance.php#L440) with this.

```php
$endpoint = '<redacted>';

$body = wp_json_encode([
    'order_number'  => $data_json->metadata->order_number,
    'status' => $data_json->status,
]);

$options = [
    'body'        => $body,
    'headers'     => [
        'Content-Type' => 'application/json',
    ],
    'timeout'     => 60,
    'redirection' => 5,
    'blocking'    => true,
    'httpversion' => '1.0',
    'sslverify'   => false,
    'data_format' => 'body',
];

wp_remote_post( $endpoint, $options );
```

In summary:

1. Online order is made, we use WooCommerce webhooks to feed that into our CRM.
2. Online Divido application updates are fed back to WordPress
3. The above code forwards these updates to our CRM so colleagues don't need to check WordPress regarding online orders.

**Describe the solution you'd like**
A WordPress action that I can hook into. Something like:

```php
do_action( 'divido_order_status_updated' , $data_json );
```

Then I can just hook into it via functions.php and be able to update the plugin knowing I won't have to re-apply the change each time.
